### PR TITLE
Always transform response code

### PR DESCRIPTION
**Pull request by PhoenixHe-msft**

### CODE REVIEW

**Review by leni-msft — COMMENTED** [review]
Is there any test case to cover the change?

File: lib/liveValidation/liveValidator.ts [hunk 1]
```diff
@@ -495,6 +488,18 @@ export class LiveValidator {
     };
   }
 
+  private transformResponseStatusCode(liveResponse: LiveResponse) {
+    // If status code is passed as a status code string (e.g. "OK") transform it to the status code
+    // number (e.g. '200').
+    if (
+      !http.STATUS_CODES[liveResponse.statusCode] &&
+      utils.statusCodeStringToStatusCode[liveResponse.statusCode.toLowerCase()]
```
Comment (raych1) [inline lib/liveValidation/liveValidator.ts:496]: Can you add a test to validate the statusCode returned in response?

Comment (PhoenixHe-msft) [reply lib/liveValidation/liveValidator.ts:496]: Test added

**Review by PhoenixHe-msft — COMMENTED** [review]
Test added

**Review by leni-msft — APPROVED** [review]

**Review by raych1 — APPROVED** [review]

**Pull request merged**